src/components/TitlesCarousel/TitlesCarousel.js

```diff
@@ -45,8 +49,14 @@ export const MovieCarousel = async ({ movies, id }) => {
     });
 
     return `
+    ${Button({
+        onClick: () => redirect(`${route}`),
+        id: `${id}`,
+        text: `${btnName}`
```

Comment:
You are passing string here so you don't need the `${btnName}` passing btnName is enought, same for route, minor, approved for now